common/build.gradle

```diff
@@ -1,10 +1,13 @@
-
-
 plugins {
     id "org.sonarqube" version "2.5"
 }
 
 apply plugin: 'com.android.library'
+apply plugin: 'pmd'
+apply plugin: 'checkstyle'
```

Comment:
Same as the above for `pmd`, we should run `checkstyle` and have it clean. Will make an issue and link back.

Reply:
https://github.com/AzureAD/microsoft-authentication-library-common-for-android/issues/83